### ISSUE TITLE
Creating a new Conda env fails on Windows

### ISSUE THREAD

**Issue by zrisher** [open-source-economics/Tax-Calculator#1095]
If a Windows user builds a new Conda environment, they are presented a version of pandas greater than the specified working version, which makes tests fail. The reason for this is covered in #1093 and fixed by #1094.

However, when the specified versions of `pandas` and `numpy` are loaded via Conda on Windows, they no longer produce working versions. Both packages will fail to import:

```
(taxcalc-dev) c:\Sites\ospc\tax-calculator>conda list numpy
# packages in environment at C:\Dev\Miniconda3\envs\taxcalc-dev:
#
Using Anaconda Cloud api site https://api.anaconda.org
numpy                     1.10.4                   py35_2

(taxcalc-dev) c:\Sites\ospc\tax-calculator>conda list pandas
# packages in environment at C:\Dev\Miniconda3\envs\taxcalc-dev:
#
Using Anaconda Cloud api site https://api.anaconda.org
pandas 
                   0.18.0              np110py35_0
(taxcalc-dev) c:\Sites\ospc\tax-calculator>python
Python 3.5.2 |Continuum Analytics, Inc.| (default, Jul  5 2016, 11:41:13) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Dev\Miniconda3\envs\taxcalc-dev\lib\site-packages\numpy\__init__.py", line 112, in <module>
    from ._globals import ModuleDeprecationWarning, VisibleDeprecationWarning
ImportError: No module named 'numpy._globals'
>>> import pandas
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Dev\Miniconda3\envs\taxcalc-dev\lib\site-packages\pandas\__init__.py", line 37, in <module>
    import pandas.core.config_init
  File "C:\Dev\Miniconda3\envs\taxcalc-dev\lib\site-packages\pandas\core\config_init.py", line 14, in <module>
    import pandas.core.config as cf
AttributeError: module 'pandas' has no attribute 'core'
```

This appears similar to the issues I experienced with conda builds of these packages in the past on Windows, i.e. #669. Previously, updating the included package version fixed the issue, and indeed that is true here: If I update pandas and numpy to their latest versions, I'm able to import both successfully.

However, bumping the pandas version by even 0.0.1 creates some errors in our test suite, as covered in #1091.

Thus, once #1093 and #1091 are resolved, we confirm the new versions of pandas and numpy work on other systems, and the requirements are bumped, this will be fixed.

**Comment by martinholmer** [comment]
@zrisher,  Thank you very much for investigating issue #1086.  All this environment installation business is above my pay grade, so I'll leave your proposed changes to @talumbau and @MattHJensen to discuss.

But if I am understanding things correctly, it does appear as if the environment installation is not being tested on Windows.  That seems to be a problem that needs to be rectified.

**Comment by zrisher** [comment]
@martinholmer You are correct, Windows builds are not being tested via CI. 

As you probably know, Windows works in fundamentally different ways from Linux and OSX, all the way down to the kernel. As such, most low-level code that interacts with the operating system must be implemented differently, thus the difficulty in doing even basic things like environment package management (for which Conda is a god-send). 

This OS difference tends to be less of a problem for languages with big Windows communities (e.g. C and its flavors) and more of a problem where much of the community uses Linux and Mac (e.g. Ruby, Python).

This also means that Travis doesn't even support Windows testing yet (see https://github.com/travis-ci/travis-ci/issues/2104). So extending CI to Windows would take some work.

**Comment by zrisher** [comment]
Ah and I see @talumbau mentions a possible work around for the Travis issues in #1086 by using AppVeyor. Nice!

**Comment by zrisher** [comment]
@talumbau Somehow I'm still unable to build a working `numpy=1.10.4` via Conda on Windows in Python 3.4 or 3.5, even though the AppVeyor box does it just fine. I *can* install it in Python 2.7, but I'd like to stick with the CI-tested environment.

I ran through all the environment build steps from [a recent successful AppVeyor run](https://ci.appveyor.com/project/talumbau/tax-calculator/build/1.0.18), creating a matching environment at `C:\Miniconda` and completely removing and rebuilding the `taxcalc` environment. I end up with a perfectly matching `conda list` in both places, but running tests or simply importing `numpy` still produces `ImportError: No module named 'numpy._globals'`. It's pretty frustrating.

Upgrading to `numpy=1.11.2` solves this for me - it installs fine and I can pass all the tests. I have no idea why I'm running into this issue and AppVeyor doesn't when using the same environments, but if all tests pass with the newer version it seems an overall improvement.

**Comment by zrisher** [comment]
This is now solved via merged PR #1114.
